features/steps/steps.go

```diff
@@ -0,0 +1,56 @@
+package steps
+
+import (
+	"encoding/json"
+	"io"
+	"os"
+	"strings"
+
+	"github.com/cucumber/godog"
+	"github.com/stretchr/testify/assert"
+)
+
+func (c *Component) RegisterSteps(ctx *godog.ScenarioContext) {
+	c.apiFeature.RegisterSteps(ctx)
+
+	ctx.Step(`^the response body is the same as the json in "([^"]*)"$`, c.theResponseBodyIsTheSameAsTheJSONIn)
+}
+
+func (c *Component) theResponseBodyIsTheSameAsTheJSONIn(expected string) error {
+	responseBody := c.apiFeature.HttpResponse.Body
+	body, _ := io.ReadAll(responseBody)
+
+	content, err := os.ReadFile(expected)
+	if err != nil {
+		return err
+	}
+
+	str := strings.ReplaceAll(string(content), "\n", "")
+	str = strings.ReplaceAll(str, " ", "")
+
+	trimmedBody, err := removeTimeParameter(string(body))
+	if err != nil {
+		return c.StepError()
+	}
+
+	assert.Equal(c, str, strings.ReplaceAll(trimmedBody, " ", ""))
```

Comment:
This stripping of spaces seems slightly misleading - given your json files don't have spaces in, one could make the assumption that the response isn't supposed to either...but it is.

Reply:
Could Marshal these into a Go Struct.